### ISSUE TITLE
Linkify rule-numbers that don't have `SC` prepended also

### ISSUE THREAD

**Issue by TriMoon** [vscode-shellcheck/vscode-shellcheck#1558]
@felipecrs The PR https://github.com/vscode-shellcheck/vscode-shellcheck/pull/539 forgot to linkify rule-numbers that **don't** have `SC` prepended...

Although this function is already prepared to take care of that: https://github.com/vscode-shellcheck/vscode-shellcheck/blob/cd4263db3ed59e9da54825b21425e0d0b9254318/src/utils/link.ts#L1-L7

This regExp in line 34 doesn't catch them: https://github.com/vscode-shellcheck/vscode-shellcheck/blob/cd4263db3ed59e9da54825b21425e0d0b9254318/src/linkify.ts#L34
Maybe this regExp would be better suited? :thinking: 
```ts
const pattern = /\b(SC)?\d{4}\b/g;
```

Catching them would IMHO **maybe** need a small modification to account for it, in: https://github.com/vscode-shellcheck/vscode-shellcheck/blob/cd4263db3ed59e9da54825b21425e0d0b9254318/src/linkify.ts#L35-L47

Don't forget to adjust your test for it also :hand_over_mouth: 
https://github.com/vscode-shellcheck/vscode-shellcheck/blob/cd4263db3ed59e9da54825b21425e0d0b9254318/test/linkify.test.ts#L9-L21

Unfortunately I'm too busy to create a ready-to-use PR for you guys... :wink: 
:vulcan_salute:

**Comment by felipecrs** [comment]
Thank you, I'll fix it.

**Comment by TriMoon** [comment]
Thanks for the fast response, i just started using this extension and :heart: it already !
Thanks for continueing the maintenance of it, i was afraid it wasn't updated anymore :+1:
